spec/controllers/users_controller_spec.rb

```diff
@@ -42,21 +42,26 @@
           let(:inviter) { object_generator(:user) }
           let(:invitation) { object_generator(:invitation, inviter: inviter) }
         
-          before { post :create, user: { email_address: invitation.recipient_email_address, password: 'password',
-                                full_name: invitation.recipient_name}, invitation_token: invitation.token }
-        
-          it "makes the user follow the inviter" do
-            recipient = User.find_by_email_address(invitation.recipient_email_address)
-            expect(recipient.follows?(inviter)).to be true
+          it "delegates to InvitationHandler to handle invitation" do
+            handled_invitation = double("handled_invitation")                         
+            InvitationHandler.stub(:new).and_return(handled_invitation)
+            expect(handled_invitation).to receive(:handle_invitation)
+            
+            post :create, user: { email_address: invitation.recipient_email_address, password: 'password',
```

Comment:
Styling of this looks a little bit more inconsistent. I would align params in user hash at least. Spec itself looks good to me.